runtime-imaging/jxrlib/autobuild/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# from https://src.fedoraproject.org/rpms/jxrlib/blob/rawhide/f/jxrlib.spec
```

Comment:
Why did we replace a whole CMakeLists.txt?

Reply:
original project's Makefile attempt build a Java binding which is not required for now, has no way to disable and fail to build here.

a tracking repo has been created at https://github.com/AOSC-Tracking/jxrlib with this change.

Reply:
CMakeLists.txt itself is originally from https://jxrlib.codeplex.com/workitem/13 (archive at https://codeplexarchive.org/codeplex/projecttab/issues/jxrlib/13), which says:

> # Patch for warnigns + CMake for building
> The following changes were made for the Fedora package which I'm proposing upstream:
> 
> - jxrlib_warnings.patch: Fix various warnings 
> - CMakeLists.txt: use CMake for building, to facilitate the creation of shared libraries, as well as cross-compilation. Debian does similarly.
>